test/sharness/t0250-files-api.sh

```diff
@@ -186,6 +186,12 @@ test_files_api() {
     test_cmp ls_l_expected ls_l_actual
   '
 
+  test_expect_success "file has correct hash and size listed with -l" '
+    echo "file1	$FILE1	4" > ls_l_expected &&
+    ipfs files ls -l /cats/file1 > ls_l_actual &&
+    test_cmp ls_l_expected ls_l_actual
+  '
+
```

Comment:
Please add a test for the ls output without `-l` is unchanged when used on a file.